_component/tabs/_support.md

```diff
@@ -1,3 +1,3 @@
-| Feature       | Chrome | Firefox | Internet Explorer | Safari |
-|---------------|--------|---------|-------------------|--------|
-| Basic Support | Latest | Latest  | 10+               | 5.1+   |
+| Feature       | Chrome | Firefox | Internet Explorer | Edge   | Safari |
+|---------------|--------|---------|-------------------|--------|--------|
+| Basic Support | Latest | Latest  | 11                | Latest | Latest |
```

Comment:
Let's make a note around here about this support being for the processed code or the standalone version since the post CSS and ES6 aren't supported in all these places

Reply:
Updated the browser support section to clarify that IE11 does not support ES6 `@import`